src/java.desktop/share/classes/javax/swing/plaf/basic/BasicScrollBarUI.java

```diff
@@ -1613,7 +1614,15 @@ public void actionPerformed(ActionEvent e) {
             // and mouseReleased is not called, then timer will not be stopped
             // Stop the timer if frame is disabled
             Component parent = scrollbar.getParent();
+            Component focusOwner = FocusManager.getCurrentManager().getFocusOwner();
             do {
+                // If application isn't in focus, stop the timer
+                if (focusOwner == null) {
+                    ((Timer)e.getSource()).stop();
+                    buttonListener.handledEvent = false;
+                    scrollbar.setValueIsAdjusting(false);
+                    return;
+                }
```

Comment:
Can't it be merged with below frame disable logic as it does the same thing?

Reply:
I initially separated it to make it clear that this is for focus. I'll edit this and modify the existing comment to include null focus.

Reply:
I tested different ways to merge this block with the below blocks. I'm not sure if it makes sense to do so now because it wouldn't be as simple as adding `|| focusOwner == null` to the first if-statement or appending the logic to the `else` block. I would still need another if or else/if block regardless. And I think it's harder to follow than the current implementation. Let me know if you prefer it a different way, but I think the current implementation is better for now.

Reply:
I was thinking more of `if (!par.isEnabled() !! focusOwner == null)` as the parent is still JFrame.

But, I was trying to run the testcase mentioned in the JBS and this regression test but I am not able to reproduce without the fix..
It doesn't keep on scrolling when it regains focus in my windows 11 machine..

Reply:
@prsadhuk  I wasn't able to replicate the issue first few times. Then changed the no. of combobox items to 1000 and I was able to reproduce. 

I think the issue with 100 combobox items is the end of combobox is reached quickly while trying to ALT +TAB at the same time and it becomes difficult to replicate the issue unless it is done quickly.

@DamonGuy Maybe it is good to increase the no. of combobox items to reproduce the issue more consistently.

Reply:
I did update the test to use 1000 items instead. However, the automated test that Alexander provided seems to work well with 100 as well. I see no issues in testing with 100, so I left it at 100 again.